__tests__/app.test.js

```diff
@@ -95,4 +95,64 @@ describe('/api/articles', () => {
       })
     
   })
+describe('POST: api/articles/:article_id/comments', () => {
```

Comment:
I'd also like to see a test to check what happens if we send a comment with an unnecessary property 🙂